open-banking-brasil-financial-api-1_ID1-ptbr.md

```diff
@@ -176,108 +176,107 @@ Para efeitos deste documento, os termos definidos em [RFC6749], [RFC6750], [RFC7
 
-If the cnpj Claim is requested as an Essential Claim for the ID Token or UserInfo response with a
-values parameter requesting a specific cnpj value, the Authorization Server MUST return an cnpj
-Claim Value that contains a **set** of CNPJs one of which must match the requested value. If this
- is an Essential Claim and the requirement cannot be met, then the Authorization Server MUST treat
- that outcome as a failed authentication attempt.
+**CNPJ**, abreviação de Cadastro Nacional de Pessoas Jurídicas, é um número de identificação de empresas **brasileiras** emitidas pelo Ministério da Fazenda **brasileira**, **na** "Secretaria da Receita Federal" ou "Ministério da Fazenda" do Brasil. No modelo de identidade do Open Banking Brasil, pessoas físicas podem se associar a 0 ou mais CNPJs. Um CNPJ é uma string que consiste em números de 14 dígitos e pode começar com 0, os primeiros oito dígitos identificam a empresa, os quatro dígitos após a barra identificam a filial ou subsidiária ("0001" padrão para a sede), e os dois últimos são dígitos de soma de verificação. Para este perfil, o pedido de cnpj deve ser solicitado e fornecido como o número de 14 dígitos.
+
+Se o parâmetro `claims` **cnpj** for solicitado como essencial (`"essential":true`) para o ID token ou para a resposta ao endpoint de UserInfo e na solicitação constar no parâmetro `value` determinado **CNPJ** exigido, o Authorization Server **deve** retornar no atrbuto **cnpj** um **conjunto** de *CNPJ* relacionado com o usuário um dos quais deve incluir valor que corresponda ao da solicitação. Se for uma `claim` indicada como essencial e não puder ser preenchido ou validado (a conta não é PJ ou o CNPJ é divergente do informado no _value_, por exemplo), o Authorization Server deve tratar a solicitação como uma tentativa de autenticação com falha.
 
-Se a reivindicação cnpj for solicitada como uma reivindicação essencial para o token de ID ou resposta UserInfo com um parâmetros de valores solicitando um valor cnpj específico, o Servidor de Autorização DEVE retornar um cnpj Claim Value que contém um **conjunto** de CNPJs, um dos quais deve corresponder ao valor solicitado. Se este é uma Reivindicação Essencial e o requisito não pode ser atendido, então o Servidor de Autorização DEVE tratar esse resultado como uma tentativa de autenticação falhada.
+Se o parâmetro `claims` **cnpj** for solicitado como essencial (`"essential"=true`) o Authorization Server deve incluir no ID Token ou na resposta ao endpoint UserInfo um **conjunto** que inclua um elemento com o número do **CNPJ** relacionado à conta utilizada na autenticação do usuário na transmissora.
```

Comment:
Same comment as above, essentail:true has been redefined. Twice; one of which is incorrect. Don't redefine how to do this in the specification. The normative examples MUST be referred too.

Reply:
Ok, Your point (normative example) is about the term MUST? As we don´t have a reference to BCP14/RFC2119, I suggest that change.. In portuguyese we have difficult to express the diference bw must (external requirement) and shall... we use "deve" to both meanings.

Reply:
Sorry, my point about normative examples / references is that they are the example that must be followed. Redefining examples should be avoided however what you could do us flesh out more appendix items with more non normative examples as guides. Or add more guides and sections to the aspsp user guides.